src/src.qbs

```diff
@@ -6,6 +6,12 @@ DynamicLibrary {
     cpp.includePaths: ["./"]
     cpp.defines: ["LOG_PREFIX=\"[QSqlMigrator]\"",
                   "_BUILDING_QSQLMIGRATOR_DLL"]
+    Properties {
+        condition: qbs.toolchain.contains("clang")
+        cpp.commonCompilerFlags: "-Wno-deprecated-copy"
```

Comment:
I guess this should be exported here, so all Libs and Applications that depend on QSqlMigrator automatically inherit this flag?